rasa/shared/core/trackers.py

```diff
@@ -809,6 +813,42 @@ def latest_action_name(self) -> Optional[Text]:
             ACTION_TEXT
         )
 
+    def persist(self, path: Union[Text, pathlib.Path], filename: Text) -> Text:
+        """Saves the tracker to a file.
+
+        Args:
+            path: Directory that the tracker should be saved in.
+            filename: Name of the file to save the tracker in.
+
+        Returns:
+            Full path of the stored tracker file.
+        """
+        tracker_file = pathlib.Path(path) / filename
+        rasa.shared.utils.io.create_directory_for_file(tracker_file)
+
+        # noinspection PyTypeChecker
+        rasa.shared.utils.io.write_text_file(str(jsonpickle.encode(self)), tracker_file)
+
+        return str(tracker_file)
+
+    @classmethod
+    def load_tracker(
+        cls: Type[rasa.utils.common.T], file_path: Union[Text, pathlib.Path]
+    ) -> rasa.utils.common.T:
+        """Loads a tracker from a file.
+
+        Args:
+            file_path: Full path and name of the file.
+
+        Returns:
+            The loaded DialogueStateTracker.
+        """
+        tracker_file = pathlib.Path(file_path)
+        if not tracker_file.is_file():
+            raise FileNotFoundError(f"Could not load tracker from '{file_path}'.")
```

Comment:
I think we should use a `RasaException` here. You might want to create a new one in https://github.com/RasaHQ/rasa/blob/master/rasa/exceptions.py